backend/app/tests/utils/test_db.py

```
@@ -332,3 +422,83 @@ def test_database_connection():
     except Exception as e:
         logger.warning(f"Database connection test failed: {e}")
         raise AssertionError(f"数据库连接测试失败: {e}")
+
+
+def create_test_data():
+    """在测试数据库中创建基础测试数据。
+    
+    此函数会在测试数据库中创建测试所需的基础数据，特别是 prompts 相关测试所需的数据。
+    """
+    try:
+        from sqlmodel import Session, SQLModel, create_engine
```

Ruff (0.11.9)

`sqlmodel.SQLModel` imported but unused

Remove unused import: `sqlmodel.SQLModel`

(F401)